src/sentry/static/sentry/app/components/actions/actionLink.tsx

```diff
@@ -1,21 +1,19 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-
+import styled from 'react-emotion';
 import classNames from 'classnames';
 
 import Confirm from 'app/components/confirm';
 
-export default class ActionLink extends React.Component {
-  static propTypes = {
```

Comment:
Should we retain the propTypes for JSX usage of this component?

Reply:
Sure. I've added it back in.